client/src/ts/dd_main.ts

```diff
@@ -186,11 +186,16 @@ ws.onmessage = (event) => {
   const msg = JSON.parse(event.data);
 
   switch (msg.id) {
+    case canDefs.CanMessage.CAN_MESSAGE_BPS_HEARTBEAT:
+      console.log(event.data);
+      break;
     case canDefs.CanMessage.CAN_MESSAGE_SOLAR_DATA_REAR:  // Not working.
-      // solarReadout.value(msg.data);
+      const solarRearPower = msg.data.voltage * msg.data.current;
```

Comment:
Would you need to perform any unit conversions/scaling? If so, where are you planning on doing that?

You should probably check this, but IIRC, the firmware does not send in terms of Volts/Amps units.

Reply:
I don't think there's any explicit documentation anywhere. You can check the firmware for solar sense and see what units it uses there.

Reply:
I've asked Ali; he says that the solar sense telemetry data is not tested but should be in milliamps and millivolts. I've updated the PR to reflect that.